backend/spec/controllers/spree/admin/orders_controller_spec.rb

```diff
@@ -30,7 +30,7 @@
     let(:adjustments) { double('adjustments') }
 
     before do
-      allow(Spree::Order).to receive_messages(find_by_number!: order)
+      allow_any_instance_of(Spree::Order::ActiveRecord_Relation).to receive_messages(find_by!: order)
       allow(order).to receive_messages(contents: Spree::OrderContents.new(order))
```

Comment:
We prefer using `receive_message_chain` here as we don't want to test internal active record implementation details. This specs will fail, if Rails renames it's relation class.